src/Sylius/Bundle/CoreBundle/EventListener/CatalogPromotionEventListener.php

```diff
@@ -29,25 +28,11 @@ public function __construct(MessageBusInterface $eventBus)
         $this->eventBus = $eventBus;
     }
 
-    public function postPersist(LifecycleEventArgs $args): void
+    public function update(GenericEvent $event): void
```

Comment:
As it can be used for both creation and update, maybe just `__invoke()`? Of `dispatchCatalogPromotionUpdateEvent`? 🤷